README.md

```diff
@@ -61,10 +61,42 @@ We also provide tools to review the graphs you create and run in ttnn.
 
 <img src="./docs/source/_static/add.svg" alt="ttnn tracer example" height="250"/>
 
-## Running out-of-the-box models
+## Running performant out-of-the-box models
 
 We have working demos for models such as [ResNet](./models/demos/resnet), [BERT](./models/demos/bert), and Falcon, both [7B in ttnn](./models/demos/ttnn_falcon7b) and [40B](./models/demos/falcon40b).
 
+These are some performance metrics for our models running on Grayskull (GS). We constantly improve these
+and publish these metrics on [GitHub
+Actions](https://github.com/tenstorrent-metal/tt-metal/actions/workflows/perf-models.yaml).
+
+| Model                           | Batch size          | Current GS throughput [1] | Target GS throughput [1] |
+|---------------------------------|---------------------|---------------------------|--------------------------|
+| ResNet 50 (fps)                 | 20                  | 2070                      | 7000                     |
```

Comment:
@tt-rkim we need have 3 perf columns: 1) current  end2end 2) current device OPs only , 3) final   target

Reply:
BERT device OPs are 397 currently

Reply:
@TT-BrianLiu  need device perf for Falcon-7 decode

Reply:
@arakhmati and @eyonland in this table we probably also want another column that says "ttnn or ttlib"